src/olympia/accounts/tests/test_views.py

```diff
@@ -126,38 +123,67 @@ def test_two_users_exist(self):
 class TestRenderErrorHTML(TestCase):
 
     def make_request(self):
-        return APIRequestFactory().get(reverse('accounts.authenticate'))
+        request = APIRequestFactory().get(reverse('accounts.authenticate'))
+        request.user = AnonymousUser()
+        return self.enable_messages(request)
 
-    def login_error_url(self, **params):
+    def login_url(self, **params):
         return urlparams(reverse('users.login'), **params)
 
-    def render_error(self, error, next_path=None):
+    def migrate_url(self, **params):
+        return absolutify(urlparams(reverse('users.migrate'), **params))
+
+    def render_error(self, request, error, next_path=None):
```

Comment:
Yeah, I originally didn't accept a `request` here then I did that then all the tests needed it so it came out.